fastpair/__init__.py

```diff
@@ -10,4 +10,10 @@
 # Copyright (c) 2002-2015, David Eppstein
 # Licensed under the MIT Licence (http://opensource.org/licenses/MIT).
 
+import contextlib
+from importlib.metadata import PackageNotFoundError, version
+
 from .base import FastPair
+
+with contextlib.suppress(PackageNotFoundError):
```

Comment:
Why suppress here?

Reply:
There is absolutely a reason, and I have of course forgotten exactly what it was 😅 . Let me do some research to provide a concrete justification.

Reply:
@carsonfarmer 

So the long-short of it is that `setuptools_scm` is great at being a lightweight, streamlined, & automated VCS when compared to all the needed upkeep with a tool like [`versioneer`](https://github.com/python-versioneer/python-versioneer). Of course there are drawbacks. One is the comment I made when I opened this PR:

> after each tag/release developers must reinstall for consistent versioning

Another is that there is *potential* for mussiness in [conflicting versions](https://stackoverflow.com/a/76206192/8692996) and other edge cases with `pyproject.toml` & `sdist`. Having that safeguard allows for the user to import and use the package even if the (uncommon) mussiness occurs.

Moreover, the reason for not using `try/except` is due to [ruff: SIM105](https://docs.astral.sh/ruff/rules/suppressible-exception/), which we will be enforcing in a subsequent PR.

The pattern we have here:

```python
import contextlib
from importlib.metadata import PackageNotFoundError, version

...

with contextlib.suppress(PackageNotFoundError):
    __version__ = version(<PACKAGE>)
```

is what we have adopted across the PySAL ecosystem (where this modern infrastructure is used – e.g. [`libpysal`](https://github.com/pysal/libpysal/blob/5bda115749e6c8a57323fe86d74487dff3878a4c/libpysal/__init__.py#L24C1-L30C38))

cc @martinfleis